ENVIRONMENT VARIABLES:
	b	(1, 2)

SYSTEM VARIABLES:
	r	(1, 2)

FORMULA:
ASSUMPTION:
    INITIAL
	  (b=1)
    SAFETY
	  [](b=1)
    LIVENESS
	  []<>(b=1)
GUARANTEE:
    INITIAL
	  (r=1)
    SAFETY
	  [](r = 1)
    LIVENESS
	  []<>(r=1)

------
rb_moore_plus_one_EA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_moore_plus_one_AE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_moore_plus_one_AA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 425, in is_realizable
    assert sys_init == aut.true
AssertionError

------
rb_moore_plus_one_EE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 433, in is_realizable
    assert env_init == aut.true
AssertionError

------
rb_mealy_plus_one_EA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_mealy_plus_one_AE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_mealy_plus_one_AA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 425, in is_realizable
    assert sys_init == aut.true
AssertionError

------
rb_mealy_plus_one_EE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 433, in is_realizable
    assert env_init == aut.true
AssertionError

------
rb_moore_EA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_moore_AE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_moore_AA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 425, in is_realizable
    assert sys_init == aut.true
AssertionError

------
rb_moore_EE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 433, in is_realizable
    assert env_init == aut.true
AssertionError

------
rb_mealy_EA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_mealy_AE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/tulip-1.4.0-py3.8.egg/tulip/transys/mathset.py", line 772, in __setitem__
    raise ValueError(msg)
ValueError: key: b, cannot be assigned value: 0
Admissible values are:
	{1, 2}

------
rb_mealy_AA
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 425, in is_realizable
    assert sys_init == aut.true
AssertionError

------
rb_mealy_EE
Traceback (most recent call last):
  File "/Users/kimiahassibi/Desktop/Caltech/SURF2023/env/lib/python3.8/site-packages/omega/games/gr1.py", line 433, in is_realizable
    assert env_init == aut.true
AssertionError
